### DISCUSSION TITLE
new permissions

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#217] [Show and tell]
I'm introducing the capability for a permission to be the same as another permission, so that you specify only once, and any change will be reflected.
Opinions?

<img width="627" alt="image" src="https://user-images.githubusercontent.com/1367199/232726780-6d1c2262-7640-478f-923c-9f60b15512b2.png">

**Comment by GeorgeGeorgeGe** [comment]
Good Idea

**Comment by xrivo1** [comment]
Ottimo. sto provando ora la beta 44. per ora tutto ok
